src/index.js

```diff
@@ -786,6 +786,10 @@ if (config.gapLimit) {
   walletUtils.setGapLimit(config.gapLimit);
 }
 
-app.listen(config.http_port, config.http_bind_address, () => {
-  console.log(`Listening on ${config.http_bind_address}:${config.http_port}...`);
-});
+if (process.env.NODE_ENV !== 'test') {
```

Comment:
I fell we should move the `app` creation to another file and import it here. So, the tests can easily create a new app just importing from this new file. Maybe we should do it in a separate PR.

Reply:
Noted here https://github.com/HathorNetwork/hathor-wallet-headless/issues/89 to be done later.